schema/kind.go

```diff
@@ -87,8 +87,8 @@ const (
 	JSONKind
 )
 
-// MAX_VALID_KIND is the maximum valid kind value.
-const MAX_VALID_KIND = JSONKind
+// MaxValidKind is the maximum valid kind value.
+const MaxValidKind = JSONKind
```

Comment:
Is this change necessary? I know some go style guides suggest only camel case, but also there was no linting error and snake case helps distinguish this from the other Kind consts